packages/strapi-generate-plugin/files/admin/src/containers/App/index.js

```diff
@@ -1,3 +1,44 @@
+/////////////////////////////////////////////////////
+// DISPLAY YOUR PLUGIN IN THE ADMINISTRATION PANEL //
+/////////////////////////////////////////////////////
+/*
+1. Check the installed plugin in your package.json you will need to require them manually
+2. Copy the following code in my-app/admin/src/plugins.js (don't forget to add the other installed plugins)
```

Comment:
It would be better to simply add the local plugin in the generate plugins.js file during the build command. pretty easy to do and the user won't have to think about it. it would be equivalent to extending a plugin

Reply:
You are right.